packages/fuel-wallet/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@fuel-connectors/fuel-wallet",
+  "types": "./src/index.ts",
+  "exports": {
```

Comment:
Can we turn this into private? And release just the umbrella package? I think would require to just change the dependencies to devDependencies on the umbrella package for tsup to include in the build.

Reply:
sure 
https://github.com/FuelLabs/fuel-connectors/pull/1/commits/80845bc51669cbdf5549999e7e3cee6a2f8810f7

```tsx
// umbrella package
"dependencies": {
    "@fuel-connectors/fuel-development-wallet": "workspace:*",
    "@fuel-connectors/fuel-wallet": "workspace:*",
    "@fuel-connectors/fuelet-wallet": "workspace:*"
},
```